packages/gatsby/src/bootstrap/index.ts

```diff
@@ -51,8 +51,9 @@ export async function bootstrap(
 
   if (process.env.GATSBY_EXPERIMENTAL_PARALLEL_QUERY_RUNNING) {
     const directory = slash(context.store.getState().program.directory)
+    const program = context.store.getState().program
```

Comment:
just a little bit smaller :)
```suggestion
    const program = context.store.getState().program
    const directory = slash(program.directory)
```